site/src/components/layout.md

```diff
@@ -136,6 +140,76 @@ The specification for this layout is as follows.
+
+How do I apply it? `layout-sidecar-left` to the `layout` element.
+Required elements: all except `layout-body-hero` and `layout-body-aside` (see allowed elements).
+Allowed elements: all except `layout-body-aside`.
+
+The following block is arranged from narrow widths on the left to wider widths on the right.
+
+```txt
+  ┌────────────┐   ┌──────────────┐ ┌──────────────────────┐
+  │ Header     │   │Header        │ │ Header               │
+  ├────────────┤   ├──────────────┤ ├──────────────────────┤
+  │ Hero       │   │Hero          │ │ Hero                 │
+  ├────────────┤   ├─────┬────────┤ ├──────┬───────────────┤
+  │ Menu       │   │Menu │ Main   │ │ Menu │ Main          │
+  ├────────────┤   │     │        │ │      │               │
+  │ Main       │   │     │        │ │      │               │
+  │            │   │     │        │ │      │               │
+  │            │   │     │        │ │      │               │
+  ├────────────┤   ├─────┴────────┤ ├──────┴───────────────┤
+  │ Footer     │   │Footer        │ │ Footer               │
+  └────────────┘   └──────────────┘ └──────────────────────┘
+```
+
+The specification for this layout is as follows.
+
+| Screensize       | Behavior                                                                                                                         |
+| ---------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| Narrow           | All elements are stacked.                                                                                                        |
+| Tablet - desktop | Menu and main are side by side. Main is wider than menu.                                                                         |
+| Widescreen       | Same as tablet-desktop with scaling gutter that keeps combined width of menu and main to the width of the widescreen breakpoint. |
+
+### Sidecar right layout
+
+The "sidecar" remains as defined in the section above, but in `layout-sidecar-left` the sidecar refers to the `layout-body-aside` element, which sits to the right of `layout-body-main` on tablet screens and wider. Unlike other layouts, this layout does not allow the usage of the `layout-body-menu` containers.
+
+How do I apply it? `layout-sidecar-right` to the `layout` element.
+Required elements: all except `layout-body-hero` and `layout-body-menu` (see allowed elements).
+Allowed elements: all except `layout-body-menu`.
+
+The following block is arranged from narrow widths on the left to wider widths on the right.
+
+```txt
+ ┌──────────────┐ ┌──────────────────────┐ ┌──────────────────────┐
```

Comment:
Below it says `Narrow all elements are stacked` but this layout diagram shows side x side, on narrow does main stack atop aside?

Reply:
Fixed